packages/flow-designer/src/components/node/AbstractNode.component.tsx

```diff
@@ -93,6 +93,7 @@ type Props = {
 	onDrag?: (event: any) => void;
 	onDragEnd?: (event: any) => void;
 	onClick?: React.MouseEventHandler;
+	onDblClick?: React.MouseEventHandler;
```

Comment:
```suggestion
	onDoubleClick?: React.MouseEventHandler;
```
Shouldn't we stick to the naming used here? https://github.com/Talend/ui/pull/4082/files#diff-a210057ec27e3fe2f0ec335fd46c269a3daff6dd86ceed96089b391f49961316R231